packages/builder/src/components/common/bindings/EvaluationSidePanel.svelte

```diff
@@ -45,11 +48,11 @@
   }
 
   const copy = () => {
-    let clipboardVal = expressionResult.result
+    let clipboardVal = expressionResult
```

Comment:
What that broken, having the `.result`?

Reply:
It doesn't seem to ever actually be passed in like this - this is actually the reason why I started the process of converting - I couldn't see where the `.result` was coming from.